src/common/state/AnnotationObject.code

```diff
@@ -164,6 +164,8 @@ AnnotationObject::ProcessOldVersions(DataNode *parentNode,
         ProcessOldLine3D(searchNode);
     else if (node->AsString() == "Line2D")
         ProcessOldLine2D(searchNode);
+    else if (node->AsString() == "Text3D")
+        ProcessOldText3D(searchNode);
```

Comment:
I see a new call is added here to `ProcessOldText3D()` but am I missing something? Where in this update is that (apparently new) function defined?

Reply:
The function is already defined in the .code file. The if-test was mistakenly left out when the function was created.